web/src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -156,8 +180,9 @@ export default {
         return;
       }
 
-      // Create the download link in itemDownloads for each item in items
+      // Create download and delete links in local data for each item in items
       this.itemDownloads = {};
+      this.itemDeletes = {};
```

Comment:
I think this is a reimplementation of https://github.com/foxbenjaminfox/vue-async-computed#recalculation. You can probably keep the original `asyncComputed` and just add some kind of hook to recalculate it.

Reply:
Oh excellent. I'll revert the commit and do this in its place.

Reply:
I cut that commit out of the commit stream, and used a better approach in dfd267457867743c3c74e59568569f4c3a5adafc.